options/locale/locale_de-DE.ini

```diff
@@ -1115,6 +1115,7 @@ settings.slack_domain=Domain
 settings.slack_channel=Kanal
 settings.add_discord_hook_desc=<a href="%s">Discord</a>-Integration zu deinem Repository hinzufügen.
 settings.add_dingtalk_hook_desc=<a href="%s">Dingtalk</a>-Integration zu deinem Repository hinzufügen.
+settings.add_typetalk_hook_desc=<a href="%s">Typetalk</a>-Integration zu deinem Repository hinzufügen.
```

Comment:
Don't change translation files except english file.

Reply:
@lunny Ok! I'll updated it.

Reply:
@lunny I updated it! could you review it again?